src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/caseworker/event/CaseworkerCaseNote.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.reform.adoption.adoptioncase.caseworker.event;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.reform.adoption.adoptioncase.caseworker.event.page.AddCaseNote;
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.CaseData;
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.State;
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.UserRole;
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.access.Permissions;
+import uk.gov.hmcts.reform.adoption.common.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.reform.adoption.common.ccd.PageBuilder;
+
+@Slf4j
```

Comment:
Add comments at class and method level in next commit

Reply:
Added JavaDoc comments at Class Level